src/main/wdl/cloudsql.wdl

```diff
@@ -134,3 +134,38 @@ task DeleteDatabase {
       memory: memory
     }
 }
+
+task CsqlQuery { 
+    parameter_meta {
+        apiProjectId: { description: "The project ID of the API we will be using (note: can be different than the instance project ID)" }
+        credentials: { description: "Optional JSON credential file" }
+        queryConfig: { description: "CsqlConfig object containing database connection details and query" }
+    }
+    input {
+        String? apiProjectId
+        File? credentials
+        CsqlConfig queryConfig
+
+        Int cpu = 1
+        String memory = "128 MB"
+        String dockerImage = "wdl-kit:1.3.0"
+    }
+
+
+    command {
+        csql ${"--project_id=" + apiProjectId} ${"--credentials=" + credentials} query ~{write_json(queryConfig)}
+    }
+
+    output {
+        File stdout = stdout()
+        File stderr = stderr()
+    }
+
+    runtime {
+        docker: dockerImage
+        cpu: cpu
+        memory: memory
+        zones: queryConfig.region
```

Comment:
Setting the zone here, doen't work. It prints a warning while running miniwdl saying -'ignored runtime settings :: keys: ["zones"]'
But if you create your instance in west1 zone, the docker container always seem to be created in west1 zone so far when I tested in dev project. 
cc @jmesterh

Reply:
this is only applicable if use are using cromwell server to run the WDL + the google backend selected in cromwell , so no. But its fine to assume that most things will be in the same zone.